src/pages/Lending/Pool/components/Repay/RepayConfirm.tsx

```diff
@@ -290,7 +290,7 @@ export const RepayConfirm = ({
       memo: supportedEvmChainIds.includes(
         fromAssetId(repaymentAsset.assetId).chainId as KnownChainIds,
       )
-        ? utils.hexlify(utils.toUtf8Bytes(confirmedQuote.quoteMemo))
+        ? toHex(stringToBytes(confirmedQuote.quoteMemo))
```

Comment:
https://viem.sh/docs/utilities/toHex#usage

```suggestion
        ? toHex(confirmedQuote.quoteMemo)
```